dxr/plugins/js/__init__.py

```diff
@@ -0,0 +1,30 @@
+"""JS plugin: analyze Javascript files by building a symbol map on parse by
+esprima.
+"""
+
+from dxr.plugins import Plugin, filters_from_namespace, refs_from_namespace
+from dxr.filters import LINE
+from dxr.indexers import QUALIFIED_LINE_NEEDLE
+from dxr.plugins.js.indexers import _TreeToIndex
```

Comment:
Nope, can't import a _private_name from another module, even our own. :-) Generally I like to `import dxr.indexers` and explicitly subclass `dxr.indexers.TreeToIndex` in the other file.